code/modules/client/client procs.dm

```diff
@@ -264,6 +264,14 @@ var/next_external_rsc = 0
 
 //send resources to the client. It's here in its own proc so we can move it around easiliy if need be
 /client/proc/send_resources()
+
+	spawn
+		// Preload the HTML interface. This needs to be done due to BYOND bug http://www.byond.com/forum/?post=1487244
+		var/datum/html_interface/hi
+		for (var/type in typesof(/datum/html_interface))
+			hi = new type(null)
+			hi.sendResources(src)
+

```

Comment:
Oh. That issue is hidden for security. It's marked as resolved, but I don't think it is yet (I need to report it again, but for that I'll need a clear test case).

The issue specifies that browse_rsc works asynchronously rather than synchronously, so you can't use it in your normal flow of operations but have to preload resources first.

i.e., if you call browse_rsc() with a css file and call browse() immediately afterward, it won't find the css file because it's still being downloaded by the client. It's not meant to do that.